hbase-common/src/main/resources/hbase-default.xml

```diff
@@ -1946,4 +1946,10 @@ possible configurations would overwhelm and obscure the important.
       responses with complete data.
     </description>
   </property>
+  <property>
+    <name>hbase.master.metafixer.max.merge.count</name>
+    <value>10</value>
+    <description>Maximum regions to merge at a time when we fix overlaps noted in
+      CJ consistency report, but avoid merging 100 regions in one go!</description>
```

Comment:
How about

> The maximum regions to merge in one operation when fixing overlaps noted in the CatalogJanitor consistency report.

Reply:
![image](https://user-images.githubusercontent.com/24363196/79406633-2d65f780-7fca-11ea-9bc7-5a0d6a1eff75.png)

this description that I reference the calculateMerges method of MetaFixer

Thanks